js-core-1/week-3/lesson.md

```diff
@@ -32,7 +28,7 @@ You can check this by starting a node console in your terminal.
 
 ```sh
 $ node
-> var arr = [1, 2, 3];
+> let arr = [1, 2, 3];
```

Comment:
I'm still not really sure why we are using `let`?

Reply:
I thought it was because it's the latest syntax and enforces more control over scope and

Reply:
@jamesgorrie I'm not sure if you are referring to using `const` and `let` vs `var`, or using `const` vs `let`. About the first one, we talked about starting using ES6+ features as soon as possible. About the second one, I could have used `const` all along even if we're mutating the array, since what needs to remain constant is just the reference to it. I just thought that the fact that we can mutate an array declared using `const` could be confusing for students. But now that I think about it, I think it's ok to learn that ASAP (plus I introduced this concept last week anyway).